drivers/uktty/pl011/Config.uk

```diff
@@ -1,14 +1,12 @@
 menuconfig LIBUKTTY_PL011
 	bool "PrimeCell UART (PL011)"
 	depends on ARCH_ARM_64
+	select LIBUKBOOT
+	select LIBUKOFW
 
 if LIBUKTTY_PL011
+
 menuconfig LIBUKTTY_PL011_EARLY_CONSOLE
 	bool "Early console"
```

Comment:
Here also the two questions similar to the ones to ns16550:
- I think we should provide a help description to this option (describing how it works and how the device is detected via fdt/kernel cmdline)?
- Can we introduce the `lib/ukboot` dependency only when this option is enabled. Otherwise we don't need `lib/ukboot` for the driver, correct?

Reply:
> I think we should provide a help description to this option (describing how it works and how the device is detected via fdt/kernel cmdline)?

Answered in https://github.com/unikraft/unikraft/pull/1473#discussion_r1726161076

> Can we introduce the lib/ukboot dependency only when this option is enabled. Otherwise we don't need lib/ukboot for the driver, correct?

Correct 👍🏼